routes/api/contacts.js

```diff
@@ -22,16 +23,16 @@ router.get("/", async (req, res, next) => {
   }
 });
 
-router.get("/", async (req, res, next) => {
+router.get("/:contactId", auth, async (req, res, next) => {
   try {
-    const contact = await contacts.getContactById(req.params.id);
+    const contact = await contacts.getContactById(req.params.contactId);
```

Comment:
Teraz musisz używać dwóch kluczy przy filtrowaniu collection. Pierwszy to id kontaktu a drugi to klucz owner. Bo chodzi o to żeby każdy użytkownik miał swoje kontakty.